nearcore/src/metrics.rs

```diff
@@ -0,0 +1,22 @@
+use near_metrics::{try_create_histogram_vec, HistogramVec};
+use once_cell::sync::Lazy;
+
+pub static APPLY_CHUNK_DELAY: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_apply_chunk_delay",
+        "Time process a chunk. Tgas used by the chunk is a metric label.",
+        &["tgas_ceiling"],
+        Some(prometheus::linear_buckets(0.0, 0.05, 50).unwrap()),
+    )
+    .unwrap()
+});
+
+pub static SECS_PER_THOUSAND_TGAS: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_execution_secs_per_thousand_tgas",
```

Comment:
```suggestion
        "near_execution_seconds_per_petagas",
```

maybe?

---

[Metric&Label naming best practices](https://prometheus.io/docs/practices/naming/) also suggest that metrics should be using base units (which in this case would require splitting this histogram into two), but in our case computing the data in the runtime is the only way to actually get meaningful measurements out, so idk if its a good idea.

Reply:
Done